### PR TITLE
change print to sys.stdout.write and flush in benchmark_runner.py

### PR DESCRIPTION
**Pull request by russell-liu**

### CODE REVIEW

File: benchmark/benchmark_runner.py [hunk 1]
```diff
@@ -242,7 +242,8 @@ def run_kuzu(serialized_graph_path):
         process = subprocess.Popen(
             tuple(benchmark_cmd), stdout=subprocess.PIPE)
         for line in iter(process.stdout.readline, b''):
-            print(line.decode("utf-8"), end='')
+            sys.stdout.write(line.decode("utf-8"))
+            sys.stdout.flush()
```
Comment (benjaminwinger) [inline benchmark/benchmark_runner.py:246]: You can also use the [flush argument of print](https://docs.python.org/3/library/functions.html#print) to explicitly flush, which is a little more concise.

**Comment by Riolku** [comment]
Why is it sufficient to just flush? I would think that we need a lock.

File: benchmark/benchmark_runner.py (hunk 1 above)
Comment (russell-liu) [reply benchmark/benchmark_runner.py]: good idea

**Comment by russell-liu** [comment]
Any idea why MSVC CI is failing? I didn't expect my changes to have any effect on that.

**Comment by mewim** [comment]
> Why is it sufficient to just flush? I would think that we need a lock.

The benchmark queries are executed sequentially, not in parallel, so it seems to be just a printing issue, not a race condition.

**Comment by mewim** [comment]
> Any idea why MSVC CI is failing? I didn't expect my changes to have any effect on that.

The MSVC test might fail randomly. I have relaunched it.

**Review by mewim — APPROVED** [review]

**Pull request merged**